.agents/merge-workflow.sh

```
@@ -0,0 +1,417 @@
+# Main command processing
+case "${1:-status}" in
+    "status")
+        check_all_status
+        ;;
+    
+    "test")
+        echo -e "${BLUE}🧪 Running tests in all worktrees...${NC}"
+        echo "Running tests for all TODOs with worktrees..."
+        echo "================================================="
+        
+        local total_todos=$(yq eval '.todos | length' "$TODOS_YAML")
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)